src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/VacancySummariesProvider/VacancySummariesProvider.cs

```diff
@@ -64,6 +62,23 @@ public async Task<IList<VacancySummary>> GetProviderOwnedVacancySummariesByUkprn
             return await RunAggPipelineQuery(aggPipeline);
         }
 
+        public async Task<IList<TransferInfo>> GetTransferredFromProviderAsync(long ukprn)
+        {
+            var builder = Builders<VacancyTransferInfo>.Filter;
+            var filter = builder.Eq(TransferInfoUkprn, ukprn.ToString()) &
```

Comment:
are we not storing transferInfo.ukprn as a long??

Reply:
👍 Whoops! This was built against my dodgy test data! Will fix.